pallets/ddc-verification/src/lib.rs

```diff
@@ -4639,7 +4648,7 @@ pub mod pallet {
 
 	#[pallet::genesis_config]
 	pub struct GenesisConfig<T: Config> {
-		pub validators: Vec<T::AccountId>,
+		pub validators: Vec<(T::AccountId, T::AccountId)>,
```

Comment:
Please describe the meaning of the fist and the second addresses, maybe in a comment.

Reply:
Comment addressed